executor/src/test/java/org/k8loud/executor/action/command/clearStorage.http

```diff
@@ -0,0 +1,20 @@
+### pause server
+POST localhost:8080/execute
+Content-Type: application/json
+
+{
+  "collectionName": "command",
+  "actionName": "ClearStorageAction",
+  "params": {
+    "params": {
+      "host": "",
+      "port": "22",
+      "privateKey": "",
```

Comment:
hmm, I'd say that sending private key in http request isn't the most secure way but who cares.
If someone asks we could just say that it could be loaded from k8s secret

Reply:
Same thoughts, who cares for now